### ISSUE TITLE
JS/TS language service crashed 5 times in the last 5 minutes

### ISSUE THREAD

**Issue by vu-dao-93** [microsoft/vscode#213315]
Type: <b>Bug</b>

in a large repo, checkout back end forth between different branches.

*Expected result*: TypeScript service loads and shows the types when hover.

*Actual result*: TypeScript service crashes, and doesn't show types when hover.

VS Code version: Code 1.89.1 (Universal) (dc96b837cf6bb4af9cd736aa3af08cf8279f7685, 2024-05-07T05:14:24.611Z)
OS version: Darwin arm64 23.4.0
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple M1 Max (10 x 24)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|6, 5, 4|
|Memory (System)|64.00GB (14.60GB free)|
|Process Argv|--crash-reporter-id 5c44e19a-e9b0-46b5-a0d1-2d6e716609a9|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (24)</summary>

Extension|Author (truncated)|Version
---|---|---
jest-snippets|and|1.9.1
vscode-tailwindcss|bra|0.10.5
npm-intellisense|chr|1.4.5
vscode-eslint|dba|2.4.4
es7-react-js-snippets|dsz|4.4.3
prettier-vscode|esb|10.4.0
copilot|Git|1.194.0
copilot-chat|Git|0.15.2
vscode-github-actions|git|0.26.2
vscode-pull-request-github|Git|0.88.1
go|gol|0.41.4
svg|joc|1.5.3
jsdoc|lll|1.0.9
ts-error-translator|mat|0.10.1
remote-containers|ms-|0.362.0
vsliveshare|ms-|1.0.5918
color-highlight|nau|2.8.0
vscode-jest|Ort|6.2.5
webgl-glsl-editor|rac|1.3.8
vscode-yaml|red|1.14.0
vscode-fileutils|sle|3.10.3
shader|sle|1.1.5
gitblame|wad|11.0.0
markdown-all-in-one|yzh|3.6.2

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscorecescf:30445987
vscod805cf:30301675
binariesv615:30325510
vsaa593:30376534
py29gd2263:31024239
c4g48928:30535728
azure-dev_surveyone:30548225
a9j8j154:30646983
962ge761:30959799
pythongtdpath:30769146
welcomedialogc:30910334
pythonidxpt:30866567
pythonnoceb:30805159
asynctok:30898717
pythontestfixt:30902429
pythonregdiag2:30936856
pythonmypyd1:30879173
pythoncet0:30885854
2e7ec940:31000449
pythontbext0:30879054
accentitlementst:30995554
dsvsc016:30899300
dsvsc017:30899301
dsvsc018:30899302
cppperfnew:31000557
dsvsc020:30976470
pythonait:31006305
chatpanelc:31048052
dsvsc021:30996838
jg8ic977:31013176
pythoncenvptcf:31049071
a69g1124:31046351
pythonprt:31056678
dwnewjupytercf:31046870
26j00206:31048877

```

</details>

**Comment by mjbvz** [comment]
Does this reproduce in the [latest VS Code insiders build](https://code.visualstudio.com/insiders/) with all [extensions disabled](https://github.com/microsoft/vscode/wiki/Performance-Issues#run-with-extensions-disabled)?

**Comment by vu-dao-93** [comment]
No. I tested with the VS Code insiders build without extensions and the service loads successfully after a while.

**Comment by mjbvz** [comment]
Thanks for checking. This has already been fixed then or is caused by one of the extensions you have installed on the stable build
